model/clusters_mgmt/v1/aws_infrastructure_access_role_resource.model

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright (c) 2020 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Manages a specific aws infrastructure access role.
+resource AWSInfrastructureAccessRole {
+	// Retrieves the details of the aws infrastructure access role.
+	method Get {
+		out Body 
```

Comment:
This should contain the direction, the name and the type of the parameter: `out Body AWSInfrastructureAccessRole`